### ISSUE TITLE
ngt 2.3.6 build failure

### ISSUE THREAD

**Issue by daeho-ro** [yahoojapan/NGT#175]
Hello, we got an error during build as follows:

```
cd /tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/build/lib/NGT && /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-11  -I/tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/build/lib/NGT -I/tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib -I/tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/build/lib -fopenmp -std=gnu++11 -fPIC -Ofast -march=native -DNDEBUG -Wall -MD -MT lib/NGT/CMakeFiles/ngtstatic.dir/Graph.cpp.o -MF CMakeFiles/ngtstatic.dir/Graph.cpp.o.d -o CMakeFiles/ngtstatic.dir/Graph.cpp.o -c /tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib/NGT/Graph.cpp
In file included from /tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib/NGT/ObjectSpace.h:19,
                 from /tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib/NGT/ObjectSpaceRepository.h:27,
                 from /tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib/NGT/Graph.h:23,
                 from /tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib/NGT/Graph.cpp:18:
/tmp/ngt-20241217-5472-yvx2hk/NGT-2.3.6/lib/NGT/PrimitiveComparator.h:23:10: fatal error: NGT/PrimitiveComparatorNoArch.h: No such file or directory
   23 | #include "NGT/PrimitiveComparatorNoArch.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

and indeed, there is no file for that. Should we change build process or are there any way to fix this? 

Thanks,

**Comment by masajiro** [comment]
I have released v2.3.7 instead of v2.3.6 to include the necessary files.

**Comment by daeho-ro** [comment]
Thank you, it works!
